Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:14.04
+FROM ruby:2.1
+
+MAINTAINER tech@execonline.com
+
+RUN apt-get update && apt-get install -y \
+  rubygems-integration \
+  git-core \
+  curl \
+  vim
+
+# Set an environment variable to store where the app is installed to inside
+# of the Docker image.
+ENV INSTALL_PATH /myapp
+RUN mkdir -p $INSTALL_PATH
+
+# This sets the context of where commands will be ran in and is documented
+# on Docker's website extensively.
+WORKDIR $INSTALL_PATH
+
+RUN mkdir ~/.ssh
+RUN ssh-keyscan -t rsa github.com > ~/.ssh/known_hosts
```

Comment:
this is from platform docker file. i think it had to do with the fact that in one of our gem files we had a git:// path. but it probably isn't needed for this repo